app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/broadcastreceiver/LocaleChangedBroadcastReceiver.kt

```diff
@@ -0,0 +1,16 @@
+package io.github.droidkaigi.confsched2018.presentation.common.broadcastreceiver
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import io.github.droidkaigi.confsched2018.util.initNotificationChannel
+import timber.log.Timber
+
+class LocaleChangedBroadcastReceiver : BroadcastReceiver() {
+    override fun onReceive(context: Context?, intent: Intent?) {
+        Timber.d("LocaleChangedBroadcastReceiver.onReceive")
+        if (Intent.ACTION_LOCALE_CHANGED == intent!!.action) {
```

Comment:
Probably we can use `intent?.action`. What do you think?

Reply:
Oh, it is my shallowness...
I fix soon, so wait please a little.

Reply:
Fixed!